src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImpl.java

```diff
@@ -631,25 +742,28 @@ else if ( NoteLine.contains(CuDisbursementVoucherConstants.DV_EXTRACT_NOTE_PREFI
                                     if (NoteLine.substring(0,2).contains(CuDisbursementVoucherConstants.DV_EXTRACT_TYPED_NOTE_PREFIX_IDENTIFIER)) {
                                         // Trim the first two characters from the note and assign it as the first user typed note line
                                         NoteLine = NoteLine.substring(2);
-                                        FirstNoteAfterAddressInfo = (NoteLine.length() <= 72) ? NoteLine : NoteLine.substring(0,72);
-                                        
+                                        stripLeadingSpace(NoteLine);
+                                        SecondNoteAfterAddressInfo = stripLeadingSpace(obtainNoteLineSectionExceedingCheckStubLine(NoteLine, FirstNoteAfterAddressInfo));
+                                        FirstNoteAfterAddressInfo = stripTrailingSpace(FirstNoteAfterAddressInfo.concat(obtainLeadingNoteLineSection(NoteLine, FirstNoteAfterAddressInfo)));
```

Comment:
Just to confirm, does a space need to be inserted manually when building the final version of the first line here? Or is one the strings guaranteed to have a leading or trailing space at this point?